versions/latest/modules/en/pages/rancher-admin/scc-registration.adoc

```diff
@@ -0,0 +1,107 @@
+= SUSE Customer Center Registration
+:revdate: 2025-08-19
+:page-revdate: {revdate}
+
+Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI or through Helm. This will enable you to see your registered systems in SCC under your subscription. 
```

Comment:
```suggestion
Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI or through Helm. This enables you to see your registered systems in SCC under your subscription. 
```